api/lib/application/sco-organization-learners/index.js

```diff
@@ -204,6 +204,43 @@ const register = async function (server) {
         tags: ['api', 'sco-organization-learners'],
       },
     },
+    {
+      method: 'PUT',
+      path: '/api/sco-organization-learners/passwords',
```

Comment:
Étant donné qu'il ne s'agit pas de la mise à jour d'une ressource qui serait `passwords` mais plutôt d'une API pour workflow c'est plutôt un verbe qu'il faut utiliser et avec un POST :

```suggestion
      method: 'POST',
      path: '/api/sco-organization-learners/password-reset',
```

L'action est *password-reset* et on lui passe de 1 à n identifiants, _passwords-reset_ au pluriel serait moins adapté.

Reply:
c'est bon pour moi. Je remets l'article que tu m'as partagé pour des infos si ça intéresse d'autres personnes : https://www.thoughtworks.com/insights/blog/rest-api-design-resource-modeling 😄